tests/test_mtaosCsc.py

```diff
@@ -448,7 +448,7 @@ async def test_run_wep_comcam(self):
                 data_path=self.data_path,
                 ofc_data=ofc_data,
                 run_name=self.run_name,
-                collections="LSSTCam/calib/unbounded,LSSTCam/raw/all",
+                collections="refcats/gen2,LSSTCam/calib,LSSTCam/raw/all",
```

Comment:
yeah.. me too.. but this is how ts_wep is accessing the catalogs in their test.. And since the MTAOS uses that same dataset, I had to use it as well..